src/styles/drawer.css

```diff
@@ -74,6 +74,25 @@
   display: flex;
 }
 
+[role=banner] .coz-drawer--apps .coz-drawer--apps-paginator {
+  display: flex;
+  flex-direction: row;
+  overflow: scroll;
+  padding-bottom: 1em;
+}
+
+[role=banner] .coz-drawer--apps .coz-drawer--apps-paginator ul {
+  min-width: 100%;
+}
+
+[role=banner] .coz-drawer--apps ul li {
+  flex-basis: calc(100% / 3);
```

Comment:
You should write `flex: 0 0 calc(100% /3);` 
From what i can recall, every browsers don't have the same default values for `flex-grow` and `flex-shrink`